lib/backend/dynamo/dynamodbbk_test.go

```diff
@@ -51,6 +52,7 @@ func (s *DynamoDBSuite) SetUpSuite(c *check.C) {
 		return New(context.Background(), map[string]interface{}{
 			"table_name":         s.tableName,
 			"poll_stream_period": 300 * time.Millisecond,
+			"region":             "us-east-2",
```

Comment:
Any reason why you used `us-east-2` vs the `us-west-2` above here? Ideally we'd not hardcode a region at all (even for testing) but I guess the tests don't work without it?

Reply:
They didn't; but I added this at the "thrashing around trying to get things to work" stage. Will try the tests without and see what happens.

Reply:
So yeah, removing region from here and adding it to my `~/.aws/config` appears to work. Will add a note in the `README`.

Reply:
Done.

Reply:
Thank you!